packages/api/db/migration/20201124123610_crop_farm_field_traceability.js

```diff
@@ -10,6 +10,7 @@ const tables = [
   'pesticide',
   'shift',
   'taskType',
+  'fieldWorkLog',
```

Comment:
@anant00729 

This looks like a migration from 2020. Is it okay to edit old migrations? I think this could have negative consequences.

Reply:
Yes I can see how it would work fine to rebuild database and work as expected. But why in the other migration file (alter_filed_work_task_table) did you use `alterTable()`?  Could you have gone back to the original table creation migration too and make the table correctly in an older file?  If we did this we could probably condense down all the migrations into one or two files.

> The process of adding a column is similar to creating a table. We never want to edit a migration file after it has been run because when we run knex migrate:latest knex will not make the change. Migrations will only run once.
 
> [From: http://perkframework.com/v1/guides/database-migrations-knex.html](
http://perkframework.com/v1/guides/database-migrations-knex.html)

Reply:
I tested it using the latest dump of the beta database. it looks like to migration only ran for the newly added files under the migration folder. Therefore, I had to remove this. In the migration file 20221114230005_alter_field_work_task_table.js

I have added these Base Model related columns

Reply:
ie. 
created_by_user_id
updated_by_user_id
created_at
updated_at